Python 2.7.7 (default, Jun  1 2014, 14:17:13) [MSC v.1500 32 bit (Intel)] on win32
Type "copyright", "credits" or "license()" for more information.
>>> int(3)
3
>>> 3.14*3
9.42
>>> 3.14*3*3
28.259999999999998
>>> (1+3)/2
2
>>> 5%3
2
>>> 2**2
4
>>> 2**6
64
>>> 5/9*(75-36)
0
>>> 5.0/9.0(75-36)

Traceback (most recent call last):
  File "<pyshell#8>", line 1, in <module>
    5.0/9.0(75-36)
TypeError: 'float' object is not callable
>>> 5.0/9.0*(75-36)
21.666666666666668
>>> 5.0/9*(75-36)
21.666666666666668
>>> 5/9.0*(75-36)
21.666666666666668
>>> 10%7
3
>>> #引入模块
>>> import module_name

Traceback (most recent call last):
  File "<pyshell#14>", line 1, in <module>
    import module_name
ImportError: No module named module_name
>>> import math
>>> dir(math)
['__doc__', '__name__', '__package__', 'acos', 'acosh', 'asin', 'asinh', 'atan', 'atan2', 'atanh', 'ceil', 'copysign', 'cos', 'cosh', 'degrees', 'e', 'erf', 'erfc', 'exp', 'expm1', 'fabs', 'factorial', 'floor', 'fmod', 'frexp', 'fsum', 'gamma', 'hypot', 'isinf', 'isnan', 'ldexp', 'lgamma', 'log', 'log10', 'log1p', 'modf', 'pi', 'pow', 'radians', 'sin', 'sinh', 'sqrt', 'tan', 'tanh', 'trunc']
>>> #math
>>> math.pi
3.141592653589793
>>> math.sin
<built-in function sin>
>>> math.sin(3)
0.1411200080598672
>>> help(sin)

Traceback (most recent call last):
  File "<pyshell#21>", line 1, in <module>
    help(sin)
NameError: name 'sin' is not defined
>>> help(math.sin)
Help on built-in function sin in module math:

sin(...)
    sin(x)
    
    Return the sine of x (measured in radians).

>>> #关系运算符
>>> 10==12
False
>>> 10<>12
True
>>> #<>与！=一样
>>> #逻辑运算符
>>> 12||12
SyntaxError: invalid syntax
>>> 12 and 12
12
>>> 12 and 11
11
>>> 12 or 11
12
>>> (2000 or 12)
2000
>>> 123 and 456
456
>>> 12 and 0
0
>>> # and 输出后面的
>>> # 正确解释
>>> #and语句中，从左至右演算表达式的值，碰到第一个假值（默认0，（），[]，{}……等为假值）则返回此假值；若全为真值，则返回最后一个真值；
or语句中，从左至右演算表达式的值，碰到第一个真值，则返回此值；若全为假值，则返回最后一个假值。
SyntaxError: invalid syntax
>>> pi=3.14
>>> 
